src/gluonts/dataset/rolling_dataset.py

```diff
@@ -214,6 +227,7 @@ def generate_rolling_dataset(
         for window in strategy.get_windows(prediction_window):
             new_item = item.copy()
             new_item["target"] = np.concatenate([base, window.to_numpy()])
```

Comment:
Can we replace the `target` string with the corresponding `FieldName`?

Reply:
Changed it.